### DISCUSSION TITLE
Possible issue in the tutorial documentation.

### DISCUSSION THREAD

**Discussion by bobdobbs** [wpengine/faustjs#785] [Q&A] [answered]
I'm walking through the tutorial material in the docs. I'm looking at [this page]: (https://faustjs.org/docs/tutorial/first-react-component)

"Finally, we'll create a list of posts by looping through the posts array and rendering a Post component for each post:"

If I copypaste the code underneath that text into my-app/src/pages/posts.tsx, while the app is running then after I hit save I get this message in the browser:

```
./src/pages/posts.tsx:2:0
Module not found: Can't resolve 'components/post'
  1 | // https://faustjs.org/docs/tutorial/first-react-component
> 2 | import Post from 'components/post';
  3 | 
  4 | 
  5 | const posts = [

https://nextjs.org/docs/messages/module-not-found
```

**Comment by blakewilson** [comment, marked as answer]
Hey @bobdobbs,

make sure you have [created the `post.tsx` component](https://faustjs.org/docs/tutorial/first-react-component#create-a-post-component) that lives in `components/post`.

You'll also want to make sure you've [setup TypeScript appropriately,](https://faustjs.org/docs/tutorial/basic-headless-site#setup-typescript) as this is how the compiler knows the base path for "components/post".
